src/code_80057C60.c

```diff
@@ -6653,7 +6693,7 @@ void func_8006D474(Player* player, s8 playerId, s8 screenId) {
         }
     }
     if ((gModeSelection == BATTLE) && (player->unk_002 & (2 << (screenId * 4)))) {
-        func_8006BA94(player, playerId, screenId);
+        render_player_balloons(player, playerId, screenId);
```

Comment:
Battle balloons?
Don't we already have a render_battle_balloons?

maybe this should be like attach_battle_balloons_to_player or something like that?

Reply:
no longer change it